ts/session/apis/open_group_api/opengroupV2/ApiUtil.ts

```diff
@@ -142,36 +142,33 @@ const defaultServerPublicKey = 'a03c383cf63c3c4efe67acc52112a6dd734b3a946b9545f4
 const defaultRoom = `${defaultServer}/main?public_key=${defaultServerPublicKey}`;
 
 const loadDefaultRoomsSingle = () =>
-  allowOnlyOneAtATime(
-    'loadDefaultRoomsSingle',
-    async (): Promise<Array<OpenGroupV2InfoJoinable>> => {
-      const roomInfos = parseOpenGroupV2(defaultRoom);
-      if (roomInfos) {
-        try {
-          const roomsGot = await getAllRoomInfos(roomInfos);
-
-          if (!roomsGot) {
-            return [];
-          }
-
-          return roomsGot.map(room => {
-            return {
-              ...room,
-              completeUrl: getCompleteUrlFromRoom({
-                serverUrl: roomInfos.serverUrl,
-                serverPublicKey: roomInfos.serverPublicKey,
-                roomId: room.id,
-              }),
-            };
-          });
-        } catch (e) {
-          window?.log?.warn('loadDefaultRoomloadDefaultRoomssIfNeeded failed', e);
+  allowOnlyOneAtATime('loadDefaultRoomsSingle', async () => {
```

Comment:
We have lost the Promise return typing on the async function?

Reply:
it's a generic now and correctly typed ! thanks to https://type-level-typescript.com/